ci-operator/config/openshift/sriov-network-operator/openshift-sriov-network-operator-master.yaml

```diff
@@ -11,6 +11,10 @@ base_images:
     name: "4.15"
     namespace: ocp
     tag: openstack-installer
+  operator-index:
+    name: redhat-operator-index
+    namespace: ci
+    tag: v4.10
```

Comment:
should you change the tag? this points to master should it be 4.15

Reply:
no, I checked other operators and they all use 4.10 for some reason. I left it like this for consistency.

Reply:
sounds good thanks! +1